.github/workflows/stale.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           # Exempt all issues with milestones from being marked as stale.
           exempt-all-issue-milestones: true
 
-          exempt-issue-labels: 'discussion 💬,pinned 📌,enhancement 🦄 ⭐,beginner friendly,maintainer action required ❕'
+          exempt-issue-labels: 'discussion 💬,pinned 📌,enhancement 🦄 ⭐,beginner friendly,maintainer action required❕,on hold  ✋🏽'
```

Comment:
Should there only be one space in `on hold`?
```suggestion
          exempt-issue-labels: 'discussion 💬,pinned 📌,enhancement 🦄 ⭐,beginner friendly,maintainer action required❕,on hold ✋🏽'
```

Reply:
I copied that directly from the label, which appeared to have two spaces.  I can remove the extra space on the label and file, if you'd like.  😄